joomla/joomla-cms
# Correct javascript error/info/success messages extra spacing

#### Summary of Changes

Following the PR which removed the extra spacing in error/info/success messages in joomla core, this PR does the same for javascript error/info/success messages.
#### Testing Instructions
1. Go to Global configuration and send a test mail
2. You will get a message like this (notice the extra spacing below the text)
   ![image](https://cloud.githubusercontent.com/assets/9630530/14189281/601e4e52-f784-11e5-8823-f74d7da27b19.png)
3. Apply this patch and after clear javascript cache (Ctrl+F5)
4. Repeat step 1
5. You will get a message like this (notice there is no extra spacing below the text)
   ![image](https://cloud.githubusercontent.com/assets/9630530/14189346/af3a2150-f784-11e5-803c-61cbcb351a3f.png)
